### PR TITLE
Add tips for students article

### PR DESCRIPTION
**Pull request by Iduoad**
This PR add a new article about tips and resources for students.

### CODE REVIEW

**Review by yjose — APPROVED** [review]
Lah ya3tikk sa7a, naddi 🙌 
Just few remarks

File: articles/9-tips-for-student.md [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+Do whatever you want, as long as you move. When you move, you meet new people, expose yourself to new ideas, gain experiences, and unlock opportunities.
+
+In my fourth year of school, I started saying “yes” to everything. That year was packed with events and connections, many of which helped me find internships and my first job. 
+
+I’m not saying you have to say yes to everything (it maybe be not the healthiest approach) but as long as you’re putting yourself out there, you’re increasing your chances of landing opportunities.
+
+Opportunities follow the laws of physics: either you move toward them, or they move toward you. The bigger the opportunity, the harder it is for it to come to you 😉.
+
+#### Useful resources
+🔗 [Geeksblabla]((https://x.com/geeksblabla): Follow us on social media. We often share Moroccan tech events.
```
Comment (yjose) [inline articles/9-tips-for-student.md:50]: Let's use LinkedIn page here

File: articles/9-tips-for-student.md [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+> 💡 Tip: Mix slow learning experiences, like reading books or taking courses, with faster ones, like watching YouTube tutorials or reading blog posts. This will train your brain to adapt to different modes of learning and help you create your own style.
+
+#### Useful resources
+🔗 [Awesome Moroccan podcasts](https://awesome-morocco.dev/podcasts): List of Moroccan tech podcasts.
+🔗 [The Missing Semester](https://www.missingsemester.io/): Courses about useful topics often not thaught in school.
+🔗 [MIT Courseware](https://ocw.mit.edu/): MIT course about various tech fields.
+🔗 [Roadmap.sh](https://roadmap.sh/): Roadmap for Software Engineers with resources and projects.
+
+## 2. Keep Moving
+
```
Comment (yjose) [inline articles/9-tips-for-student.md:38]: Would love to add here, if you didn't find a community or club  in your school you can start one

**Review by yjose — APPROVED** [review]

File: articles/9-tips-for-student.md (hunk 2 above)
Comment (Iduoad) [reply articles/9-tips-for-student.md:38]: Done! Sorry for te3tila

File: articles/9-tips-for-student.md (hunk 1 above)
Comment (Iduoad) [reply articles/9-tips-for-student.md:50]: Done!

File: articles/9-tips-for-student.md [hunk 3]
```diff
@@ -0,0 +1,171 @@
...
+Start by using open-source tools. This alone is a form of contribution. Then, engage with the community: read the code, ask questions, open issues, suggest ideas, or join discussions on forums and Discord.
+
+Even without contributing a single line of code, this process will teach you how software works, help you build a name for yourself in the community, and prepare you to contribute code when you’re ready.
+
+> 💡 Tip: Building your own open-source project also counts as OSS contribution. 😉
+
+#### Useful resources
+🔗 [Open Source & Hacktoberfest 21](https://www.youtube.com/watch?v=T0ebJvUiDG4): A Geeksblabla episode about Open Source.
+🔗 [Moroccan Open Source projects](https://awesome-morocco.dev/open-source): List of Moroccan Open Source projects.
+🔗 [Open Source Guide](https://opensource.guide/): Various guides on how to contribute to open source.
```
Comment (Iduoad) [inline articles/9-tips-for-student.md:122]: Will these be rendered each per line?

Comment (yjose) [reply articles/9-tips-for-student.md:122]: I think so,  will fix them if it's not the case

**Review by yjose — APPROVED** [review]

**Comment by yjose** [comment]
@Iduoad probably you missed pushing your updates

**Pull request merged**